plansys2_executor/include/plansys2_executor/ActionExecutor.hpp

```diff
@@ -66,6 +66,7 @@ class ActionExecutor
   std::vector<std::string> get_action_params() const {return action_params_;}
   plansys2_msgs::msg::ActionExecution last_msg;
 
+  rclcpp::Time get_current_time() const {return node_->now();}
```

Comment:
Is not this function redundant, if you have a Node?

Reply:
Yes. Reverting this. See cf4a474